src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
-}
+  const styles = sourceString
+    .split(';')
+    .filter((style) => style.trim().length > 0);
```

Comment:
The filter method does not mutate the original array, so there is no need to make a copy before using it.